pages/api/v1/employees.ts

```diff
@@ -39,6 +40,10 @@ export default async function handler(
   req: NextApiRequest,
   res: NextApiResponse
 ) {
+  if (!isAuthorized({ req })) {
```

Comment:
The swagger doc for these endpoints needs the auth token info added to it (like in /deparments)